packages/@aws-cdk/aws-dynamodb-global/README.md

```diff
@@ -0,0 +1,31 @@
+## @aws-cdk/aws-dynamodb-global
+
+Here is a minimal deployable Global DynamoDB tables definition:
+
+```javascript
+import globaldynamodb = require('@aws-cdk/aws-dynamodb-global');
+const CONSTRUCT_NAME = 'aws-cdk-dynamodb-global';
+const TABLE_PARTITION_KEY: Attribute = { name: 'hashKey', type: AttributeType.String };
+const STACK_PROPS: DynamoDBGlobalStackProps = {
```

Comment:
This is not idiomatic, pass the properties directly when creating the construct:

```typescript
new GlobalTable(stack, 'aws-cdk-dynamodb-global', {
    dynamoProps: {
        partitionKey: TABLE_PARTITION_KEY,
        tableName: TABLE_NAME
    },
    regions: [ 'us-east-1', 'us-east-2', 'us-west-2' ],
    // etc.
});
```